strictdoc/export/html/generators/project_map.py

```diff
@@ -0,0 +1,23 @@
+# mypy: disable-error-code="no-untyped-def"
```

Comment:
Is it possible that you remove this mypy ignore. It is a new file, so you probably just need to add just a few types here and there.